amplify.yml

```diff
@@ -0,0 +1,23 @@
+version: 1
+backend:
+  phases:
+    build:
+      commands:
+        - npm ci --cache .npm --prefer-offline
+        - npm run generate-dependencies
+        - npx ampx pipeline-deploy --branch $AWS_BRANCH --app-id $AWS_APP_ID
+frontend:
+  phases:
+    build:
+      commands:
+        - npm run generate-dependencies
```

Comment:
if you've run it before in the backend you should not need to run it here - the 'backend' and 'frontend' stages run in the same environment

Reply:
Done.